include/dronecore.h

```diff
@@ -52,14 +52,29 @@ class DroneCore
         DEVICE_BUSY, /**< @brief %Device is busy. */
         COMMAND_DENIED, /**< @brief Command is denied. */
         DESTINATION_IP_UNKNOWN, /**< @brief %Connection IP is unknown. */
-        CONNECTIONS_EXHAUSTED /**< @brief %Connections exhausted. */
+        CONNECTIONS_EXHAUSTED, /**< @brief %Connections exhausted. */
+        URL_INVALID
```

Comment:
I think we need to be more specific `CONN_URL_INVALID` or `CONNECTION_URL_INVALID`